Error message when requests to Ensembl (liftover) fail (#5322)
Clinical-Genomics/scout: CHANGELOG.md

- Institute settings reset
- Updated color scheme for variant assessment badges that were hard to see in light mode, notably Risk Factor
- Avoid page timeout by skipping HGVS validations in ClinVar multistep submission for non-MANE transcripts from variants in build 38

> - Sashimi view page displaying an error message when Ensembl REST API (LiftOver) is not available

Reviewer: This newline "fight" is getting a little silly. Either way is fine, but conventions we stick to are just that -  conventions. They can change at any time, but then there should be reason or at least consensus?

To me, there appears to be no good criteria to determine the right or wrong in this case. The current existence of an extra typographic room around the latest unreleased adds a little clarity, but it also is not strictly necessary, and does take a second to fix on each release. It does serve as a reminder when also editing for the version to check through the unreleased changes to see that they are well written, typo free and somewhat logical to the users.

Neither PEP, GNU changelog or keepachangelog explicitly discuss whitespacing, but they also don't have an extra space for unreleased. On the other hand they use an extra newline before each subheading (added, fixed, ...), which we don't.

Let's use the energy to e.g. add PR numbers/links to the changes?

Author: Are you aware that this is your OCD speaking, right? You even went and checked the standards 🤣 
I swear to god that the day you start removing that line I will add it instead

> Let's use the energy to e.g. add PR numbers/links to the changes?
Agreed!, I'll add it to this one!

Reviewer: 😂Says the one who is making the change? 😉 Surely you also checked all available standards before taking such an audacious step on territory that is clearly difficult (as in non-quantitative, and not strongly correlated to physical entities)? But thank you for the PR number change! 👍

Author: > Surely you also checked all available standards before taking such an audacious step on territory that is clearly difficult

Of course, I would never dream of not adhering to the standards

> - Sashimi view page not crashing but displaying a red warning when Ensembl REST APU (liftover) are not available

Reviewer: ```suggestion
- Sashimi view page displaying a warning when Ensembl REST API (LiftOver) is not available

```
